services/gateway-api/src/circuit_breaker/meta_breaker.py

```
@@ -0,0 +1,357 @@
+        Args:
+            ad_account_id: Meta ad account ID
+            campaign_id: Campaign ID
+            ad_creative: Ad creative data
+            targeting: Targeting parameters
+            budget: Budget configuration
+            **kwargs: Additional parameters
+
+        Returns:
+            Response with ad ID and status
+        """
+        async def _execute():
```

Ruff (0.14.8)

Missing return type annotation for private function `_execute`

(ANN202)